HospitalBeds/R4/resources/Quesionnaire-R4-HospitalBeds.json

```diff
@@ -277,7 +277,7 @@
           "text": "Equipment",
           "type": "choice",
           "required": false,
-          "option": [
+          "answerOption": [
             {
               "valueCoding": {
                 "code": "E0250",
```

Comment:
There are bunch of different HCPCS codes in the template. Why are there only two in the questionnaire?

Reply:
The story only requires to support E0250 and E0251 :-)